### DISCUSSION TITLE
Backup & Restore

### DISCUSSION THREAD

**Discussion by yodaphone** [shlinkio/discussions#92] [Help wanted] [answered]
Hi

I'm running shlink on my local network using docker & SQLite and want to move a cloud instance. Can you please let me  if i can backup and restore ?

**Comment by acelaya** [comment, marked as answer]
Shlink does not have backup and restore per se, but it has a way to import information from different sources.

https://shlink.io/documentation/advanced/import-short-urls/

The best scenario would be if you were able to expose your current instance to your new environment. That would allow you to import from another Shlink instances, which imports both short URLs and their visits.

If that's not an option, you can still import from a CSV file, but that will import short URLs only.

First open https://app.shlink.io, connect to your existing instance, go to short URLs and click on the "export" button.

Then setup your new Shlink instance and run the import tool from it, to use whichever of the two approaches above.
